src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { Person } from '../../types';
+ 
+
+const getPreparedPerson = (people: Person[]) => {
+    return people.map(person => {
+      const mother = people.find(({ name }) => {
+        return name === person.motherName;
+      });
+      const father = people.find(({ name }) => {
+        return name === person.fatherName;
+      });
+  
+      return {
+        ...person,
+        mother,
+        father,
+      };
+    });
+  };
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(result => {
+        setPeopleList(result);
+      })
+      .catch(() => setIsError(true))
+      .finally(() => setIsLoading(false));
+
+    return () => {
+      setPeopleList([]);
```

Comment:
It doesn't make much sense, but it's a good practice to cancel your requests in case of component unmount https://plainenglish.io/community/how-to-cancel-fetch-and-axios-requests-in-react-useeffect-hook